dll/win32/uxtheme/stylemap.c

```diff
@@ -40,7 +40,11 @@ typedef struct _MSSTYLES_CLASS_MAP {
 
 typedef struct _MSSTYLES_CLASS_NAME {
     const MSSTYLES_CLASS_MAP *lpMap;
+#if __REACTOS__
```

Comment:
Please fix all these `#if __REACTOS__` to `#ifdef __REACTOS__`

Reply:
This has been done in f552e84.